src/main/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -0,0 +1,106 @@
+      respAcceptance.amountPaid = draftClaimantResponse.paidAmount.amount
+    }
+    respAcceptance.formaliseOption = this.getFormaliseOption(draftClaimantResponse.formaliseRepaymentPlan)
+    respAcceptance.determinationDecisionType = draftClaimantResponse.courtDecisionType
+    if (draftClaimantResponse.settleAdmitted && draftClaimantResponse.settleAdmitted.admitted === YesNoOption.YES) {
+      return respAcceptance
+    }
+    respAcceptance.claimantPaymentIntention = this.convertPaymentIntention(draftClaimantResponse.alternatePaymentMethod)
+    respAcceptance.courtDetermination = this.createCourtDetermination(draftClaimantResponse)
+    return respAcceptance
+  }
+
+  private static createCourtDetermination (draftClaimantResponse: DraftClaimantResponse): CourtDetermination {
+    const courtDetermination: CourtDetermination = new CourtDetermination()
+    courtDetermination.courtDecision = draftClaimantResponse.courtOfferedPaymentIntention
+    if (draftClaimantResponse.rejectionReason) {
+      courtDetermination.rejectionReason = draftClaimantResponse.rejectionReason.text
+    }
+    courtDetermination.disposableIncome = draftClaimantResponse.disposableIncome
+    return courtDetermination
+  }
+
+  private static getFormaliseOption (repaymentPlan: FormaliseRepaymentPlan): string {
+    switch (repaymentPlan.option) {
+      case FormaliseRepaymentPlanOption.SIGN_SETTLEMENT_AGREEMENT :
+        return 'SETTLEMENT'
+      case FormaliseRepaymentPlanOption.REQUEST_COUNTY_COURT_JUDGEMENT :
+        return 'CCJ'
+      default:
+        throw new Error(`Unknown formalise repayment option ${repaymentPlan.option.value}`)
+    }
+  }
+
+  private static convertPaymentIntention (draftPaymentIntention: PaymentIntentionDraft): PaymentIntention {
+    const paymentIntention: PaymentIntention = new PaymentIntention()
+    paymentIntention.paymentOption = draftPaymentIntention.paymentOption.option.value as PaymentOption
+    if (draftPaymentIntention.paymentDate) {
+      paymentIntention.paymentDate = this.convertPaymentDate(draftPaymentIntention.paymentOption,draftPaymentIntention.paymentDate)
+    }
+    if (draftPaymentIntention.paymentPlan) {
+      const repaymentPlan: RepaymentPlan = {
+        firstPaymentDate: draftPaymentIntention.paymentPlan.firstPaymentDate.toMoment(),
+        instalmentAmount: draftPaymentIntention.paymentPlan.instalmentAmount,
+        paymentSchedule: draftPaymentIntention.paymentPlan.paymentSchedule.value as PaymentSchedule
+      } as RepaymentPlan
+      paymentIntention.repaymentPlan = repaymentPlan
+    }
+    return paymentIntention
+  }
+
+  private static convertPaymentDate (paymentOption: PaymentOptionDraft, paymentDate: PaymentDate): Moment {
+    switch (paymentOption.option) {
+      case PaymentType.IMMEDIATELY:
+        return MomentFactory.currentDate().add(5, 'days')
+      case PaymentType.BY_SET_DATE:
+        return paymentDate.date.toMoment()
+      default:
+        return undefined
```

Comment:
If this is not the possible path, prefer to throw error

Reply:
done... just FYI.. this is same the existing implementation of convertPaymentDate in responseModelConverter.ts